testinfra/modules/dns.py

```diff
@@ -0,0 +1,31 @@
+# coding: utf-8
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import unicode_literals
+
+from testinfra.modules.base import Module
+
+
+class Dns(Module):
+    """Test dns record exists """
+
+    def __init__(self, record=None):
+        self.record = record
+        super(Dns, self).__init__()
+
+    @property
+    def exists(self):
+        return self.run_expect([0, 1], "host %s", self.record).rc == 0
```

Comment:
I think this module does not offer valuable feature for testinfra. But I also think a kind of dns module make sense in testinfra. Maybe we should start to see what feature offer similar software (serverspec, inspec and goss). I think we can discuss about the spec (= a list of example assertions) first and then implement it.